Menus/RadialMenu/RadialMenu.cs

```diff
@@ -128,8 +128,7 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
             }
             else
             {
-                m_RadialMenuUI.buttonInputDirection = Vector3.zero;
-                return;
+                m_RadialMenuUI.buttonInputDirection = Vector2.zero;
```

Comment:
This was seriously it?

Reply:
Wait, was this the Vector2 change, or the removing of the return that fixed it?